test/emby/1.0.14/templates/service-tcp.yaml

```diff
@@ -1,8 +1,10 @@
 {{ $svc := .Values.embyServerHttp }}
 {{ $ports := list }}
-{{ $ports = mustAppend $ports (dict "name" "emby" "port" $svc.port "nodePort" $svc.port "targetPort" "emby") }}
-{{ $ports = mustAppend $ports (dict "name" "http" "port" 80 "targetPort" "emby") }}
-{{ $ports = mustAppend $ports (dict "name" "https" "port" 443 "targetPort" "emby") }}
+{{ if .Values.hostNetwork }}
```

Comment:
Do we need the service at all when using host network?

Reply:
Only if we want other apps in the cluster accessing emby via ClusterIP.
So it wont have to use the hostnetwork to communicate.

For example, tautulli would communicate with Plex for some stats and info, having an internal channel (ClusterIP) would avoid going through all the network layers to reach it via host net.

I don't know if there are any simillar apps/use cases for emby.

Reply:
There is no downside though…

Reply:
Makes sense